airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -165,6 +168,24 @@ class TaskInstance(BaseModel):
     try_number: int
     map_index: int | None = None
 
+    max_tries: int
+    pool_slots: int
+    queue: str
+    priority_weight: int
+
+    # TODO: Task-SDK: Can we replace TaskInstanceKey with just the uuid across the codebase?
+    @property
+    def key(self) -> TaskInstanceKey:
```

Comment:
What needs this btw?

Reply:
aah removed, I was playing around with consolidating `airflow/executors/workloads.py:TaskInstance`:

https://github.com/apache/airflow/blob/d200f9a0fac6caa77ede1d70fc96bc7acebb45a8/airflow/executors/workloads.py#L42-L68